### PR TITLE
Thread recipeName through manifest and add search function.

### PR DESCRIPTION
**Pull request by shans**

### CODE REVIEW

File: runtime/manifest.js [hunk 1]
```diff
@@ -214,6 +214,9 @@ class Manifest {
   findShapeByName(name) {
     return this._find(manifest => manifest._shapes.find(shape => shape.name == name));
   }
+  findRecipeByName(name) {
```
Comment (dstockwell) [inline runtime/manifest.js]: Do we need to support multiple here, like `findParticlesByVerb` above?

Comment (shans) [reply runtime/manifest.js]: findRecipesByName? Or findRecipesByVerb?

Comment (shans) [reply runtime/manifest.js]: (the latter. Done.)

Comment (mmandlis) [reply runtime/manifest.js]: could a recipe (in the future) have both - a name and a verb that are different?

**Pull request merged**